cmd/common.go

```diff
@@ -104,6 +104,14 @@ func getNullString(flags *pflag.FlagSet, key string) null.String {
 	return null.NewString(v, flags.Changed(key))
 }
 
+func getStringSlice(flags *pflag.FlagSet, key string) []string {
```

Comment:
I wasn't sure if this was really necessary, as an empty slice is fine. I just wrote it so we could have a `panic` error handler 😃